moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -221,12 +211,6 @@ servo:
     }
   }
 
-  low_latency_mode: {
-    type: bool,
-    default_value: false,
-    description: "If true , low latency mode is enabled."
-  }
```

Comment:
We're currently using this parameter on MoveIt Studio for Gazebo, but it's kind of a workaround and I'm OK removing this parameter and seeing what happens / looking for better solutions.

Reply:
We can add this later if necessary :+1: